hybrid/detailed_pv_plant.py

```diff
@@ -124,6 +140,7 @@ def system_capacity_kw(self, size_kw: float):
         :param size_kw:
```

Comment:
can you please add to the comment whether this is DC or AC?

Reply:
Done (DC).